### PR TITLE
Release 1.14.0

### PR DESCRIPTION
**Pull request by halge**

### CODE REVIEW

File: sokannonser/settings.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import os
 import datetime
 
+API_VERSION = '1.14.0.dev'
```
Comment (oskardrenske-ith) [inline sokannonser/settings.py:4]: change to '1.14.0'

**Review by halge — COMMENTED** [review]
ok, can be merged. Need to add notes to CHANGELOG.md

File: sokannonser/settings.py (hunk 1 above)
Comment (halge) [reply sokannonser/settings.py]: fixed

File: sokannonser/repository/text_to_concept.py [hunk 2]
```diff
@@ -31,7 +30,7 @@ def __init__(self, ontologyhost='localhost', ontologyport=9200,
 
         self.ontology = None
 
-        if settings.ES_HOST != 'localhost':
+        if settings.ES_HOST != '127.0.0.1':
```
Comment (lerca) [inline sokannonser/repository/text_to_concept.py:33]: When running windows 'localhost' is localhost and not 127.0.0.1 as on Mac. So maybe we should check both to determine if localhost:
        if settings.ES_HOST != '127.0.0.1' and settings.ES_HOST != 'localhost' :

File: sokskrapade/rest/__init__.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+def lowercase_maxlength(value):
+    if not value:
+        raise ValueError('string type must be non-null')
+    if len(value) > 255:
+        raise ValueError('parameter can not be longer than 255 characters')
+
+    return str(value).lower()
+
+
+# QF_CHOICES = ['occupation', 'skill', 'location', 'employer']
```
Comment (oskardrenske-ith) [inline sokskrapade/rest/__init__.py:29]: commented-out code

File: sokskrapade/rest/skrapade.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+GROUP = 'occupation-group'
+FIELD = 'occupation-field'
+MUNICIPALITY = 'municipality'
+REGION = 'region'
+COUNTRY = 'country'
+QUERY = 'q'
+LIMIT = 'limit'
+OFFSET = 'offset'
+
+#OCCUPATION = 'occupation-name'
```
Comment (oskardrenske-ith) [inline sokskrapade/rest/skrapade.py:19]: commented-out code

File: sokskrapade/rest/__init__.py (hunk 3 above)
Comment (halge) [reply sokskrapade/rest/__init__.py:29]: i did it cos not used, but for skrapade we can leave it.

File: sokskrapade/__init__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+from flask import Flask
+from flask_cors import CORS
+from jobtech.common.customlogging import configure_logging
+from sokannonser import appconf
+from sokskrapade.rest import api
+from sokskrapade.rest.skrapade import SearchJobLink
+
+app = Flask(__name__)
+CORS(app)
+configure_logging([__name__.split('.')[0], 'sokannonser', 'jobtech'])
```
Comment (lerca) [inline sokskrapade/__init__.py:11]: sokskrapade instead of sokannonser in configure_logging

File: sokskrapade/__init__.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import logging
```
Comment (lerca) [inline sokskrapade/__init__.py:1]: Major change (to the naming of the folders, classes, etc) would be to change from sokskrapade to sokjoblinks. To scrape is just a method for fetching and cleaning data from web. Joblinks explains more what this is.

File: sokskrapade/rest/skrapade.py [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+#SKILL = 'skill'
+#PLACE = 'place'
+
+@ns_skrapade.route('joblinks')
+class SearchJobLink(Resource):
+    @ns_skrapade.doc(
+
+
+        description='Search scraped ads using parameters and/or freetext\n'
```
Comment (lerca) [inline sokskrapade/rest/skrapade.py:28]: Change from 'scraped ads' to something with 'joblinks'? Perhaps check with Jonas the official name of the api/system?

File: tests/download_all_ads_to_file.py [hunk 8]
```diff
@@ -3,7 +3,7 @@
 import requests
 
 api_key = ''
-url = "http://localhost:5000/snapshot"
+url = "http://127.0.0.1:5000/snapshot"
```
Comment (lerca) [inline tests/download_all_ads_to_file.py:6]: Is this compatible on windows?

Comment (oskardrenske-ith) [reply tests/download_all_ads_to_file.py:6]: it works on Windows. It's a utility I wrote to be able to have the ads on file for comparison with the search results.

File: sokskrapade/rest/skrapade.py (hunk 7 above)
Comment (halge) [reply sokskrapade/rest/skrapade.py:28]: we can do it later

File: sokskrapade/rest/skrapade.py (hunk 4 above)
Comment (halge) [reply sokskrapade/rest/skrapade.py:19]: ok for POC.

File: sokskrapade/__init__.py (hunk 5 above)
Comment (halge) [reply sokskrapade/__init__.py]: done,

File: sokskrapade/__init__.py (hunk 6 above)
Comment (halge) [reply sokskrapade/__init__.py:1]: later refactor

**Review by lerca — APPROVED** [review]

**Review by oskardrenske-ith — APPROVED** [review]

**Pull request merged**